source/gent/tier1_hortense.rst

```diff
@@ -436,6 +436,21 @@ A list of available partitions can be obtained using ``module avail cluster/dodr
 
 To check the currently active partition, use ``module list cluster``.
 
+Requesting GPU resources
+++++++++++++++++++++++++
+
+Don't forget to actively request GPU resources in your jobs or from the commandline.
+Only loading the cluster/dodrio/gpu_rome_a100 module is not sufficient.
+You should request 12 cores per GPU (remember: 4 GPUs per node, 48 cores per node).
```

Comment:
ok, there must be a better way. i prefer users don't have to do this math

Reply:
Okay, now I understand the comments.
I adjusted the documentation accordingly.